_locales/en/messages.json

```diff
@@ -1783,9 +1783,27 @@
     },
       "configurationVesselName": {
         "message": "Craft name"
+    },
+      "onboardLoggingBlackbox": {
+    	"message": "Blackbox logging device"
+    },
+      "onboardLoggingRateOfLogging": {
+    	"message": "Portion of PID loop iterations to log (logging rate)"  
+    },
+      "onboardLoggingSerialLogger": {
+    	"message": "Outboard serial logging device"  
+    },
+      "onboardLoggingFlashLogger": {
+    	"message": "Onboard dataflash chip"  
+    },
+      "onboardLoggingEraseInProgress": {
+    	"message": "Erase in progress, please wait..."  
     }
     
     
```

Comment:
Your editor seems to be adding extra lines at the end of the file. Maybe you can disable this?